config/i18n.json

```diff
@@ -850,6 +850,7 @@
     "All": "All",
     "CantPullFromPostmaster": "You must go to the tower to retrieve this item.",
     "CannotCurrentlyRoll": "This perk cannot be rolled on the current version of this item.",
+    "UnreliablePerkOption": "This perk appears only in the collections view, it might not actually roll on this item.",
```

Comment:
- can we pick a conjunction or make it two sentences?
- "actually" is a bit extraneous in this thought
- maybe clearer to say "might not roll randomly on this item"?